src/src_mysql/sadb_routine_mariadb.template.c

```diff
@@ -444,7 +444,7 @@ static int32_t parse_sa_from_mysql_query(char* query, SecurityAssociation_t** se
             }
             if (strcmp(field_names[i], "acs") == 0)
             {
-                sa->acs = atoi(row[i]);
+                *sa->acs = atoi(row[i]);
```

Comment:
Since this is now a byte field, we need to prepare it exactly like we do the ecs byte field.

Reply:
This should be resolved now.